### PR TITLE
[RA2 Ch03] - Tweak wording to cover both physical and virtual

### LINKED ISSUE #1850: [RA-2 Ch03] 3.2.1.5 Container Runtime Services - change requested
[3.2.1.5 Container Runtime Services](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/kubernetes/chapters/chapter03.md#3215-container-runtime-services) 

The current description is "bare metal" centric whereas minor changes could enable support for both req.inf.vir.01 and req.inf.phy.01.

For example, changing the following by deleting the indented words:
"The Container Runtime is the component that runs within a *Host* Operating System (OS) and manages the underlying OS functionality, such as cgroups and namespaces (in Linux), in order to provide a service within which container images can be executed and make use of the infrastructure resources (compute, storage, networking and other I/O devices) abstracted by the *Host* OS, based on API instructions from the kubelet."

### PR DESCRIPTION
**Pull request by tomkivlin**
Resolves #1850 

Also wrapped lines to 80 chars for future ease of reviews.

### CODE REVIEW

File: doc/ref_arch/kubernetes/chapters/chapter03.md [hunk 1]
```diff
@@ -46,108 +81,268 @@ The features and capabilities described in the software and hardware profiles ar
...
-
-For Kubernetes the important interface to consider for container management is the [Kubernetes Container Runtime Interface (CRI)](https://kubernetes.io/blog/2016/12/container-runtime-interface-cri-in-kubernetes/). This is an interface specification for any container runtime so that it is able to integrate with the kubelet on a Kubernetes Node. The CRI decouples the kubelet from the runtime that is running in the Host OS, meaning that the code required to integrate kubelet with a container runtime is not part of the kubelet itself (i.e. if a new container runtime is needed and it uses CRI, it will work with kubelet). Examples of this type of runtime include containerd (with cri plugin) and cri-o, which is built specifically to work with Kubernetes.
-
-To fullfill `req.inf.vir.01` the architecture should support a container runtime which provides the isolation of Operating System kernels.
-
-The architecture must support a way to isolate the compute resources of the infrastructure itself from the workloads compute resources.
+The Container Runtime is the component that runs within a Container Host
```
Comment (ulikleber) [inline doc/ref_arch/kubernetes/chapters/chapter03.md:159]:
```suggestion
The Container Runtime is the component that runs within a Kubernetes Node Operating System (OS) and manages the underlying OS functionality, such as cgroups and namespaces (in Linux), in order to provide a service within which container images can be executed and make use of the infrastructure resources (compute, storage, networking and other I/O devices) abstracted by the Kubernetes Node OS, based on API instructions from the kubelet.
```
I think we should consistently use Kubernetes Node.

**Review by ulikleber — APPROVED** [review]
Approved with the small change below.

File: doc/ref_arch/kubernetes/chapters/chapter03.md (hunk 1 above)
Comment (tomkivlin) [reply doc/ref_arch/kubernetes/chapters/chapter03.md]: Fixed in https://github.com/cntt-n/CNTT/pull/1894/commits/710f0d7e5289c9a7559c2d6dabb4ecc80e46c76d (single line)

File: doc/ref_arch/kubernetes/chapters/chapter03.md [hunk 2]
```diff
@@ -46,108 +81,268 @@ The features and capabilities described in the software and hardware profiles ar
...
+For Kubernetes the important interface to consider for container management is
+the [Kubernetes Container Runtime Interface
+(CRI)](https://kubernetes.io/blog/2016/12/container-runtime-interface-cri-in-kubernetes/).
+This is an interface specification for any container runtime so that it is able
+to integrate with the kubelet on a Kubernetes Node. The CRI decouples the
+kubelet from the runtime that is running in the Host OS, meaning that the code
+required to integrate kubelet with a container runtime is not part of the
+kubelet itself (i.e. if a new container runtime is needed and it uses CRI, it
+will work with kubelet). Examples of this type of runtime include containerd
+(with cri plugin) and cri-o, which is built specifically to work with
```
Comment (xmulligan) [inline doc/ref_arch/kubernetes/chapters/chapter03.md:200]:
```suggestion
(with CRI plugin) and cri-o, which is built specifically to work with
```

File: doc/ref_arch/kubernetes/chapters/chapter03.md [hunk 3]
```diff
@@ -46,108 +81,268 @@ The features and capabilities described in the software and hardware profiles ar
...
+interfaces to the pods, the CNTT architecture must support a CNI metaplugin/CNI
+multiplexer.
+
+A CNI metaplugin/CNI multiplexer has the capability to attach several
+interfaces, using different other CNI plugins, to a pod. Note that the different
+network characteristics of the interfaces might require different networking
+technologies, which would potentially require different CNI plugins.
+
+To comply with `req.inf.ntw.08`, inter node communication must be served by a
+CNI plugin which complies with the default K8s networking assumptions.
```
Comment (xmulligan) [inline doc/ref_arch/kubernetes/chapters/chapter03.md:225]:
```suggestion
To comply with `req.inf.ntw.08`, inter node communication must be served by a
CNI plugin which complies with the default [Kubernetes networking assumptions](https://kubernetes.io/docs/concepts/cluster-administration/networking/#the-kubernetes-network-model).
```

**Review by walterkozlowski — APPROVED** [review]

**Comment by tomkivlin** [comment]
@xmulligan I've updated as per your comments, are you OK to approve?

**Review by xmulligan — APPROVED** [review]

**Comment by tomkivlin** [comment]
@scottsteinbrueck this is ready for your code owner review/merge.  I'll merge in 48 hours otherwise.

**Review by petorre — APPROVED** [review]

**Pull request merged**